js/main.js

```diff
@@ -20,4 +20,77 @@ $entryForm.addEventListener('submit', function (event) {
   data.entries.unshift(submittedData);
   $image.setAttribute('src', 'images/placeholder-image-square.jpg');
   $entryForm.reset();
+
+  const newData = renderEntry(submittedData);
```

Comment:
Should have a `$` before the variable, because this is technically storing a DOM element